modules/sandboxed-containers-deploy-node-check-feature.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying-sandboxed-container-workloads.adoc
+
+:_module-type: PROCEDURE
+[id="sandboxed-containers-deploy-node-check-feature_{context}"]
+= Deploying the check node feasibility feature
+
+Before you can check whether a cluster node is eligible to run sandboxed containers, you need to deploy the check node feasibility for Kata containers feature in your cluster.
```

Comment:
We might want to let users know at this point (or perhaps at modules/sandboxed-containers-check-nodes.adoc above, line 10?) that they have two other options (AFAIK - if nothing changed since I last looked):
- do nothing which will result in sandboxed runtime being installed on all workers.  This would be reasonable if a user is sure due to external knowledge that all workers are eligible.
- add the feature.node.kubernetes.io/runtime.kata label manually to nodes that they want to have the sandboxed runtime installed on.  This gives users an opportunity to control precisely which workers get kata installed.

Reply:
Thanks! Actually broke the procedure out into three separate tasks, let me know if you think this works.

Reply:
Hmm - on consideration, need to make a few more changes.